lib/xapi/null_problem.rb

```diff
@@ -0,0 +1,62 @@
+module Xapi
+  class NullProblem
+    attr_reader :track_id, :slug, :path, :language
+
+    def initialize(attributes)
+      attributes.each {|field, value|
+        instance_variable_set(:"@#{field}", value)
+      }
+    end
+
+    def name
```

Comment:
Let's not make a fancy name for it if we don't have it. Just alias this to the slug.